apps/conte/src/app/components/dashboard/treatment-plans/treatmentplans.component.ts

```diff
@@ -1,13 +1,16 @@
 import { Component, OnInit } from '@angular/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
 import { TreatmentPlanService } from '../../../Shared/services/treatmentPlan.service';
+import { GenericModalComponent } from '../../../Shared/components/modals/generic/generic-modal.component';
 
 @Component({
   selector: 'conte-treatmentplans',
   templateUrl: './treatmentplans.component.html',
   styleUrls: ['./treatmentplans.component.scss'],
 })
 export class TreatmentplansComponent implements OnInit {
-  treatmentplans: any = [];
+  treatmentPlans: any = [];
+  modal: any;
```

Comment:
give it a proper name according to it's function for example, a modal that updates task should be taskUpdateModalRef,
as it is not an actual modal but it's reference thus the 'Ref' at the end.